tv_apps/fling-tutorial/offline_content/js/index.js

```diff
@@ -0,0 +1,24 @@
+/* global FTEWizard
+*/
+'use strict';
+(function(exports) {
+
+  var tutorialContainer = document.body;
+
+  var tutorial = new FTEWizard('castingTutorial');
+  tutorial.init({
+    container: tutorialContainer,
+    pageClass: 'slide',
+    buttonsClass: 'slide-button',
+    launchEveryTime: true,
+    propagateKeyEvent: true,
+    onfinish: function () {
+      // Rewite the location to trigger the mozbrowserlocationchange event
+      // to let the outer app know user finishes this tutorial
+      window.location.href = window.location.href + '#finished';
+    }.bind(tutorial)
+  });
+
+  exports.tutorial = tutorial;
```

Comment:
I did some investigation but still can't find the specific reason why the button is not focused. Maybe you can try to listen to it on some event like `load.